cms/envs/common.py

```diff
@@ -1015,6 +1015,11 @@
 ##### EMBARGO #####
 EMBARGO_SITE_REDIRECT_URL = None
 
+##### custom vendor plugin variables #####
+ADDITIONAL_NODE_ENV_VARS = {
+    'SAMPLE_PLUGIN_VARIABLE': 'sampleValue'
```

Comment:
An empty dict should suffice here - this feels antipattern to the file

Reply:
:+1: